### ISSUE TITLE
Add RDS Support

### ISSUE THREAD

**Issue by Ghe69** [gbolo/aws-power-toggle#63]
Hi,

It would be nice to have RDS also supported.
Thank you.

**Comment by gbolo** [comment]
Hi @Ghe69. Do you mean the ability to toggle RDS related EC2 instances? I will look into that, thanks for the suggestion.

**Comment by Ghe69** [comment]
Yes that's adding the capability to start and stop a RDS Instance.

**Comment by ashish235** [comment]
+1
Adding RDS will be a great feature.
